src/hooks/useSearch.ts

```diff
@@ -0,0 +1,19 @@
+import { useState, ChangeEventHandler, FormEventHandler } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+export const useSearch = () => {
+  const [keyword, setKeyword] = useState<string>('');
```

Comment:
```suggestion
  const [keyword, setKeyword] = useState('');
```
이렇게해도 타입 추론이 될까요?? 단순 질문입니다!!